OpenSim/Simulation/Manager/Manager.h

```diff
@@ -173,11 +168,64 @@ class OSIMSIMULATION_API Manager
     { _writeToStorage =  writeToStorage; }
 
     // Integrator
+    enum class IntegratorMethod {
+        /** For details, see SimTK::CPodesIntegrator. */
+        CPodes,
+        
+        /** For details, see SimTK::ExplicitEulerIntegrator. */
+        ExplicitEuler,
+        
+        /** For details, see SimTK::RungeKutta2Integrator. */
+        RungeKutta2,
+        
+        /** For details, see SimTK::RungeKutta3Integrator. */
+        RungeKutta3,
+        
+        /** For details, see SimTK::RungeKuttaFeldbergIntegrator. */
+        RungeKuttaFeldberg,
+
+        /** For details, see SimTK::RungeKuttaMersonIntegrator. */
+        RungeKuttaMerson,
+
+        //SemiExplicitEuler, no error ctrl, requires fixed stepSize arg on construction
+
+        /** For details, see SimTK::SemiExplcitEuler2Integrator. */
+        SemiExplcitEuler2,
+
+        /** For details, see SimTK::VerletIntegrator. */
+        Verlet
+    };
+
+    /** Sets the integrator method used via IntegratorMethod enum. The 
+      * integrator will be set to its default options. For MATLAB/Python
+      * users, see `setIntegrator(const std::string&)` instead.*/
+    void setIntegratorMethod(IntegratorMethod integMethod);
+    /** Sets the integrator method used via a string. The integrator 
+      * will be set to its default options. For C++ users, see 
+      * `setIntegrator(IntegratorMethod)` instead. */
+    void setIntegratorMethod(const std::string& integMethod);
```

Comment:
What do you think about adding a Matlab or Python example to make it crystal clear that the string should contain the exact same text as the enum?